test/policies/application_policy_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
+
+describe ApplicationPolicy do
+  let(:user) { "user" }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.